nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/user/UserProperties.java

```diff
@@ -1,90 +0,0 @@
-/*
- * (C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and others.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * Contributors:
- *         Vladimir Pasquier <vpasquier@nuxeo.com>
- */
-package org.nuxeo.client.api.objects.user;
-
-import java.util.List;
-
-import com.fasterxml.jackson.annotation.JsonProperty;
-
-/**
- * @since 0.1
- */
-public class UserProperties {
```

Comment:
Why not keep this class and mark it as deprecated as of 2.2 to help people migrating (if any)?